### PR TITLE
Use missing depth argument in isAncestor method

### PR DESCRIPTION
**Pull request by seferov**

### CODE REVIEW

**Comment by seferov** [comment]
if https://github.com/KnpLabs/KnpMenu/pull/260 is merged, tests on this PR will also pass

File: .travis.yml [hunk 1]
```diff
@@ -12,14 +12,15 @@ php:
     - 5.5
     - 5.6
     - 7.0
-    - hhvm
 
 matrix:
     include:
         - php: 5.3
-          env: COMPOSER_FLAGS='--prefer-lowest --prefer-stable' SYMFONY_DEPRECATIONS_HELPER=weak
+          env: COMPOSER_FLAGS='--prefer-stable' SYMFONY_DEPRECATIONS_HELPER=weak
```
Comment (stof) [inline .travis.yml]: this change does not make sense. This job is precisely about testing the lower bound

**Comment by stof** [comment]
Please rebase this PR so that it does not contain changes in the Travis config file (which are creating conflicts btw)

**Comment by seferov** [comment]
@stof done

**Comment by stof** [comment]
Thanks for fixing this bug @seferov.

**Pull request closed without merging**